WooCommerce/WooCommerceTests/ViewRelated/CardPresentPayments/CardPresentPaymentsOnboardingUseCaseTests.swift

```diff
@@ -351,35 +351,7 @@ class CardPresentPaymentsOnboardingUseCaseTests: XCTestCase {
         let state = useCase.state
 
         // Then
-        XCTAssertEqual(state, .genericError)
-    }
-
-    func test_onboarding_returns_generic_error_when_account_is_not_eligible_for_wcplay_plugin() {
-        // Given
-        setupCountry(country: .us)
-        setupWCPayPlugin(status: .active, version: WCPayPluginVersion.minimumSupportedVersion)
-        setupPaymentGatewayAccount(accountType: WCPayAccount.self, status: .complete, isCardPresentEligible: false)
-
-        // When
-        let useCase = CardPresentPaymentsOnboardingUseCase(storageManager: storageManager, stores: stores)
-        let state = useCase.state
-
-        // Then
-        XCTAssertEqual(state, .genericError)
-    }
-
-    func test_onboarding_returns_not_completed_when_account_is_not_connected_for_wcplay_plugin() {
```

Comment:
Now superseded by test_onboarding_returns_plugin_setup_not_completed_with_no_account_for_wcplay_plugin and test_onboarding_returns_plugin_setup_not_completed_with_no_account_for_stripe_plugin (we no longer return generic error if the account setup isn't complete - we treat that as the plugin setup not completed